waspc/packages/deploy/.prettierrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Awesome job adding prettier. 🎨  We will want to make sure it plays nicely with the `eslint` config. Check output from `npx eslint src`. I am fine using whatever convention so long as they both align. Thanks! :) 👍🏻